utils/globals.ts

```diff
@@ -1,5 +1,5 @@
-import { useWindowDimensons } from './hooks';
```

Comment:
Meine prettier config funktioniert anscheinend nicht richtig 🤔

Reply:
was soll in der Zeile pretified werden?